src/coreclr/nativeaot/Runtime/FinalizerHelpers.cpp

```diff
@@ -27,7 +27,7 @@ GPTR_DECL(Thread, g_pFinalizerThread);
 CLREventStatic g_FinalizerEvent;
 CLREventStatic g_FinalizerDoneEvent;
 
-extern "C" void __cdecl ProcessFinalizers();
+EXTERN_C NATIVEAOT_API void QCALLTYPE ProcessFinalizers();
```

Comment:
Is `NATIVEAOT_API` needed for anything? I would drop it.

Reply:
Not that I know of... I can remove it.

Reply:
Done.